testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/injection/support/jpa/cdi/EnabledEnhancementManagedInjectionTestCase.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.jboss.as.test.integration.ee.injection.support.jpa.cdi;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+
+/**
+ * Tests that an entity callbacks, e.g. {@link jakarta.persistence.PostPersist}, work with Jakarta Context and Dependency
+ * Injection when the {@code jboss.as.jpa.classtransformer} property is set to {@code true}.
+ *
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
+ */
+@ApplicationScoped
+@RunWith(Arquillian.class)
+@Ignore("This test can be enabled once the default data source can be used. Currently the test methods will throw a " +
+        "NullPointerException due the jpa subsystem explicitly disabling CDI support.")
+public class EnabledEnhancementManagedInjectionTestCase extends AbstractManagedInjectionTestCase {
```

Comment:
We don't actually need this one as currently CDI is explicitly disabled if bytecode enhancement is enabled. However, I left it as a placeholder for when this might be fixed. If we feel more comfortable deleting this, then adding it later that is fine too.

Reply:
Sounds like this will be enabled to recreate the failure previously seen with the RESTEasy Contact example.  Hopefully I will be ready soon to see if this test passes with the design changes I have made to improve CDI/Persistence/app datasource/default datasource integration so users don't have to choose a subset of features to use.